nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestMergeContent.java

```diff
@@ -1153,6 +1153,248 @@ public void testMaxBinAge() throws InterruptedException {
         runner.run();
 
         runner.assertTransferCount(MergeContent.REL_MERGED, 1);
+
+        runner = TestRunners.newTestRunner(new MergeContent());
```

Comment:
why not having this in a dedicated test case for this PR which tests all the attributes EL?

Reply:
Should I only check the EL values are correctly evaluated, or also that the processor works when using the EL attributes? I found the tests redundant too, but I wasn't sure about checking only that the `evaluateExpression()` worked as expected was enough